### PR TITLE
OSX ScreenShot feature added

### PR DESCRIPTION
**Pull request by bhaveshAn**

### CODE REVIEW

**Comment by bhaveshAn** [comment]
Tested and working fine, not having dependencies either. Running using `screencapture` utility, which is a pre-installed one.  Kindly merge this.
test

**Comment by kiok46** [comment]
@bhaveshAn thanks for contributing, please add `screenshotdemo.kv` file or load kivy string using `Builder.load_string()`in the example.
Thanks

File: plyer/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
            'compass', 'email', 'filechooser', 'flash', 'gps', 'gravity',
            'gyroscope', 'irblaster', 'light', 'orientation', 'notification',
            'proximity', 'sms', 'tts', 'uniqueid', 'vibrator', 'wifi',
-           'temperature')
+           'temperature', 'screenshot')
```
Comment (phunsukwangdu) [inline plyer/__init__.py]: screenshot will come between 'proximity' and 'sms'

File: plyer/facades/__init__.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
            'Compass', 'Email', 'FileChooser', 'GPS', 'Gravity', 'Gyroscope',
            'IrBlaster', 'Light', 'Orientation', 'Notification', 'Proximity',
            'Sms', 'TTS', 'UniqueID', 'Vibrator', 'Wifi', 'Flash',
-           'Temperature')
+           'Temperature', 'ScreenShot')
```
Comment (phunsukwangdu) [inline plyer/facades/__init__.py]: same here

File: plyer/__init__.py (hunk 1 above)
Comment (bhaveshAn) [inline plyer/__init__.py]: @phunsukwangdu why, since temperature has been added after wifi.

File: plyer/facades/__init__.py (hunk 2 above)
Comment (bhaveshAn) [inline plyer/facades/__init__.py]: @phunsukwangdu I think it is not much necessary.

**Comment by bhaveshAn** [comment]
@KeyWeeUsr Please review this PR. Changed the `take_shot() `method to `take()`.

**Pull request closed without merging**